easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.76.0-GCC-10.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'EB_Boost'
+
+name = 'Boost.Python'
+version = '1.76.0'
+
+homepage = 'https://boostorg.github.io/python'
+description = """Boost.Python is a C++ library which enables seamless interoperability between C++
+ and the Python programming language."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://boostorg.jfrog.io/artifactory/main/release/%(version)s/source/']
+sources = ['boost_%s.tar.gz' % '_'.join(version.split('.'))]
+patches = ['Boost-1.71.0_fix-Python3.patch']
+checksums = [
+    '7bd7ddceec1a1dfdcbdb3e609b60d01739c38390a5f956385a12f3122049f0ca',  # boost_1_76_0.tar.gz
+    '60e3aede2f444a3855f4efed94d1de5c2887983876e0fae21f6ca5cfdc53ea96',  # Boost-1.71.0_fix-Python3.patch
+]
+
+multi_deps = {'Python': ['3.9.5']}
```

Comment:
@SIB-software For `Python` please specify it as a simple dependency. For the newer toolchains we no longer include a `Python 2.x` option (with rare exceptions).

Reply:
done

Reply:
Thanks.